charts/dev/livebook/values.yaml

```diff
@@ -19,12 +19,30 @@ service:
     ports:
       main:
         port: 8787
-        protocol: tcp
+        targetPort: 8080
+  headless:
+    enabled: true
+    primary: false
+    clusterIP: None
+    ports:
+      headless:
+        enabled: true
+        port: 8787
         targetPort: 8080
 workload:
   main:
     podSpec:
       containers:
         main:
           env:
-            LIVEBOOK_PASSWORD: livebookiscool
+            # A_ because these variables are sorted alphabetically and this one needs to come first.
+            A_POD_IP:
+              fieldRef:
+                fieldPath: status.podIP
+            LIVEBOOK_CLUSTER: '{{ printf "dns:%s-headless.%s.svc.cluster.local" (include "tc.v1.common.lib.chart.names.fullname" .) .Release.Namespace }}'
```

Comment:
svc.cluster.local is not guaranteed on every cluster.
It should work fine with $fullname-headless

Reply:
@stavros-k 
This does not work. Somehow the DNS resolver in Erlang doesn't work with just `$fullname-headless`. I'll have to revert this and use `svc.cluster.local`. Then it works at least in clusters svc.cluster.local is available.

Do you know the cases this isn't guaranteed?

Reply:
https://github.com/phoenixframework/dns_cluster/issues/10